source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -35,6 +35,13 @@ content: |
      USER        = <username>
      PASSWORD    = <password>
      DATABASE    = <dbname>
+
+     ; optional SSL configuration
+
+     SSLKEY      = </path/to/.pem-key-file>
+     SSLCERT     = </path/to/ssl-certificate>
+     SSLMODE     = DISABLE|ALLOW|PREFER|REQUIRE
+
```

Comment:
Please add the following after the code block:

`.. include:: /includes/fact-odbc-parameters.rst`